modules/apps/27-interchain-accounts/types/packet_test.go

```diff
@@ -83,11 +85,13 @@ func (suite *TypesTestSuite) TestValidateBasic() {
 	}
 }
 
-func (suite *TypesTestSuite) TestGetCallbackAddresses() {
+func (suite *TypesTestSuite) TestGetSourceCallbackAddress() {
+	expSrcCbAddr := "srcCbAddr"
```

Comment:
I think this could be a constant.
```suggestion
	const expSrcCbAddr = "srcCbAddr"
```